apps/epic-web/src/components/exercise-overlay.tsx

```diff
@@ -9,11 +9,25 @@ const ExerciseOverlay = () => {
   const {github} = module
 
   return (
-    <div className=" bg-black/30 ">
+    <div className="flex aspect-video items-center justify-center bg-gray-950 text-white dark:bg-black/20">
       {github?.repo && (
-        <div className="flex aspect-video flex-col items-center justify-center gap-5 p-3 text-center sm:hidden">
+        <div className="flex flex-col items-center space-y-5">
           <p className="font-text text-3xl font-bold">Now it’s your turn!</p>
-          <p className="">Try solving this.</p>
+          <p>
+            Clone{' '}
+            <a className="underline" href={github.repo} target="_blank" rel="noreferrer">
+              this repository
+            </a>{' '}
+            with the Epic Web Workshop App and follow instructions in the{' '}
```

Comment:
we should spend more time on these instructions and make them super clear and easy to follow.

some ideas:
- if they already have the workshop app running it'd be useful to include a link to given exercise. we need to consider ports and fields in sanity.
- more information about the workshop app
- overall copy